server/src/main/resources/db/changelog/20180430101420-add-intent-fields-to-consumer.xml

```diff
@@ -7,14 +7,61 @@
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <changeSet id="20180430101420-1" author="vrjain">
-        <comment> add-intent-fields-to-consumer</comment>
+        <comment> Add system purpose role column to cp_consumer</comment>
         <addColumn tableName="cp_consumer">
-            <column name="offering" type="VARCHAR(255)"/>
+            <column name="sp_role" type="VARCHAR(255)"/>
         </addColumn>
+    </changeSet>
+
+    <changeSet id="20180430101420-2" author="vrjain">
+        <comment> Add system purpose usage column to cp_consumer</comment>
+        <addColumn tableName="cp_consumer">
+            <column name="sp_usage" type="VARCHAR(255)"/>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="20180430101420-3" author="mstead">
+        <comment>Add system-purpose status column to cp_consumer.</comment>
+        <addColumn tableName="cp_consumer">
```

Comment:
Due to migration time, if each of the column adds in a single table is bundled in a single changeset it would likely update faster.

Reply:
I looked into this some last night. If the DB has autocommit enabled, then it doesn't matter if the operations are bundled into a single changeset. For this reason, liquibase recommends separating them since it can leave the DB in a bad state if something fails -- [See these docs](https://www.liquibase.org/documentation/changeset.html). I believe that MySQL has autocommit enabled by default.

Given this info, do we still want to bundle them?